### DISCUSSION TITLE
Does create-next-app not ask for example repo from version 9.5.1?

### DISCUSSION THREAD

**Discussion by dipunj** [vercel/next.js#15822] [Help] [answered]
Hello,
I recently updated my create-next-app script, which caused a behaviour to break. 

Earlier (in version 9.4.4) whenever I executed npx create-next-app, I would be asked
1. What is your project named?
2. Pick a template
> Default Starter App
> Example from the Next.js repo

But in the version 9.5.1, this is broken. The user is only prompted for the Q1 above, and not for Q2 as well.

I want to know whether this is intentional?

**Comment by timneutkens** [comment, marked as answer]
> I want to know whether this is intentional?

It is intentional, the examples list was too long and non-practical for most users so we removed it based on feedback.

**Reply by Timer** [reply]
@dipunj note you can still pass the `--example` flag to create a new app with a specific example!

**Comment by dipunj** [comment]
@timneutkens Thank you for clarifying.
